spec/namespaces/_core.yaml

```diff
@@ -3266,6 +3266,8 @@ components:
                     $ref: '../schemas/_core.search.yaml#/components/schemas/Suggest'
               terminated_early:
                 type: boolean
+              status:
```

Comment:
This should go into CHANGELOG in fixes as "Added missing `status` to `/_search/template` response"